packages/block-library/src/post-navigation-link/edit.js

```diff
@@ -49,8 +49,8 @@ export default function PostNavigationLinkEdit( {
 	const displayArrow = arrowMap[ arrow ];
 
 	if ( showTitle ) {
-		/* translators: Label before for next and previous post. There is a space after the colon. */
```

Comment:
It looks like it was intentional to include the space.

Reply:
Indeed. Looks like it's needed.

Fixed now and fixed the translator comment as well.